model_analyzer/triton/client/client.py

```diff
@@ -174,3 +180,20 @@ def is_server_ready(self):
         Returns true if the server is ready. Else False
         """
         return self._client.is_server_ready()
+
+    def _check_for_triton_log_errors(self, log_file):
+        if not log_file:
+            return
+
+        log_file.seek(0)
+        log_output = log_file.read()
+
+        if not type(log_output) == str:
+            log_output = log_output.decode('utf-8')
+
+        if log_output:
+            if "Unexpected argument:" in log_output:
+                error_start = log_output.find("Unexpected argument:")
+                raise TritonModelAnalyzerException(
+                    f'Error: TritonServer did not launch successfully\n\n{log_output[error_start:]}'
+                )
```

Comment:
would still be nice to have a catchall for a generic triton error. Is there a standard error string they print?

Reply:
Yes, it would. I'll file a follow on story to do this.